dev-doc/Adding_a_new_heapdump_format.md

```diff
@@ -0,0 +1,288 @@
+|  |- <Java Local> long[32] @ 0x5db1f460                                                                             |          272 |           272
+|  '- Total: 2 entries                                                                                               |              |              
+|- <Java Stack Frame> com.ibm.io.async.ResultHandler.runEventProcessingLoop(Z)V @ 0x925dd70  (ResultHandler.java:530)|          160 |         6,608
+|- <Java Stack Frame> com.ibm.io.async.ResultHandler$2.run()V @ 0x925dd84  (ResultHandler.java:905)                  |           20 |            20
+'- <Java Stack Frame> com.ibm.ws.util.ThreadPool$Worker.run()V @ 0x925dda4  (ThreadPool.java:1,550)                  |           32 |            32
+----------------------------------------------------------------------------------------------------------------------------------------------------
+```
+
+So the stack frame object is of a type of the method, and the class hierarchy
+is as follows:
+
+```
+Class                                                                                                                                |Type of Class
+-------------------------------------------------------------------------------------------------------------------------------------|--------------------
+<native memory>                                                                                                                      |<native memory type>
+|- <method>                                                                                                                          |<method type>
+|  |- sun.reflect.NativeMethodAccessorImpl.invoke0(Ljava/lang/reflect/Method;Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;|<method type>
+|  |- java.lang.Thread.sleep(J)V                                                                                                     |<method type>
+|  |- org.apache.axis2.jaxws.server.EndpointController.invokeOneWay(Lorg/apache/axis2/jaxws/server/EndpointInvocationContext;)V      |<method type>
+'- <native memory type>                                                                                                              |<native memory type>
+   '- <method type>                                                                                                                  |<native memory type>
+-------------------------------------------------------------------------------------------------------------------------------------|----------
+```
+
+So `<native memory>` is a new type, with no superclass (a sibling to
+`java.lang.Object`). Its class is `<native memory type>` not `java.lang.Class`.
+`<method>` is a new type, of class `<method type>`.
+
+## Other references
+
+- [Graduation review](http://archive.eclipse.org/projects/www/project-slides/Helios/MAT_Helios_Release.pdf)
```

Comment:
I guess you added this because the image was taken from there. Should we remove it too (if we didn't use the image) or are there other useful details for the topic? I didn't spot them going quickly through the document.